controllers/activemqartemisscaledown_controller.go

```diff
@@ -100,12 +101,7 @@ func (r *ActiveMQArtemisScaledownReconciler) Reconcile(ctx context.Context, requ
 	reqLogger.Info("====", "namespace:", namespace)
 	reqLogger.Info("====", "localOnly:", localOnly)
 
-	cfg, err := clientcmd.BuildConfigFromFlags(masterURL, kubeconfig)
-	if err != nil {
-		reqLogger.Error(err, "Error building kubeconfig: %s", err.Error())
-	}
-
-	kubeClient, err = kubernetes.NewForConfig(cfg)
+	kubeClient, err = kubernetes.NewForConfig(r.Config)
```

Comment:
it is not clear if any content from the first CR is relevant, in line with the rest of the controllers, the configuration form the manager should be sufficient but in a similar manner to the lack of service account and the lifecycle being dictated by the scaledown cr's, it is not 100% clear if there is any potential problem here.
For envTest, it is important that the config is passed in.